public/partials/io/import.html

```diff
@@ -70,13 +71,43 @@ <h5>Imported data sources</h5>
                 <button ng-click="cancelImport()">cancel</button>
 
             </div>
-            <div class="col-md-4">
-                <h5>Existing data sources</h5>
+            <h4 translate>The following elements already exist. Please select the ones you want to overwrite. The others will not be imported:</h4>
+            <br>
+            <div class="col-md-3">
+                <h5 translate>Existing Layers:</h5>
+                <br>
+                <ul>
+                    <li ng-repeat="item in importFile.layers">
+                        <div>
+                            {{item.name}}
+                        </div>
+                        <input type="checkbox" ng-model="item.replace" ng-value="boolean">
```

Comment:
The checkbox is displayed below the name. It would be better to have it in front of the name.

```
[✓] Report Name
```

Also, you can wrap the name inside a `<label>`  so the checkbox is automatically checked when clicking on the name.